packages/runtime/test-runtime-utils/src/mocks.ts

```diff
@@ -475,6 +480,11 @@ export class MockFluidDataStoreRuntime extends EventEmitter
     }
 
     public process(message: ISequencedDocumentMessage, local: boolean): void {
+        if (local) {
+            assert(this._pendingMessageCount > 0,
+                "Calls to process(local=true) must be balanced with submitMessage().");
```

Comment:
(This may not be a valid assert, depending on how the mock is used...  but I was curious to see if it would turn up anything.)